bindings/experimental/distrdf/python/DistRDF/Profiling/Flamegraph.py

```diff
@@ -0,0 +1,97 @@
+from functools import partial
+import os
+from typing import Callable, Optional
+
+from subprocess import run
+
+from DistRDF.Profiling.ErrorHandling import check_and_raise_errors
+from DistRDF.Profiling.Perf import collect_perf_data
+from DistRDF.Profiling.Base import ProfilingData, Visualization
+
+def Decorator(func:Callable, data_dir:str, perf_options:Optional[dict]):
```

Comment:
Maybe also here the naming should be changed to something more specific like `collect_data_for_flamegraph`